### ISSUE TITLE
OSGi unfriendly

### ISSUE THREAD

**Issue by anton-johansson** [bitsofinfo/hazelcast-docker-swarm-discovery-spi#23]
This artifact is OSGi unfriendly, it lacks the necessary manifest headers. Any plans on fixing this?

**Comment by bitsofinfo** [comment]
No plans. If you'd like to submit changes in a PR to make it so, please feel free.

**Comment by anton-johansson** [comment]
Allright!

I would have, but my Gradle skills are quite limited. I might give it a go if/when I have some time to get into Gradle. :)

**Comment by bitsofinfo** [comment]
As are my OSGI, never used it, this project just uses standard gradle jar compilation so there is likely a config option for the build that can make it compatible.

**Comment by anton-johansson** [comment]
Yeah, it's a Maven plugin (should work with Gradle, right?) so it should be fairly easy!
